site/docs/tooltip.md

```diff
@@ -64,6 +64,14 @@ To avoid that the tooltips groups the data, add an aggregate to the tooltip enco
 
 <div class="vl-example" data-name="bar_tooltip_aggregate"></div>
 
+## Tooltip image
+
+To display an image in a tooltip, you need to use the special field name `image`, which is an indicator to the Vega Tooltip plugin to render an image instead of displaying text in the tooltip. The image tooltip can be specified either by setting the `tooltip` property of the mark definition (as detailed above) or by passing the field *as an array* to the tooltip encoding channel, as in the example below:
```

Comment:
We need to clarify that this only works with Vega-Tooltip and add a link.

Reply:
Fixed!